docs/pages/guides/upgrading/1.0.mdx

```diff
@@ -0,0 +1,169 @@
+---
+meta:
+  title: "Upgrading to 1.0"
+  parentTitle: "Guides"
+  description: "Guide to upgrade to Liveblocks version 1.0"
+---
+
+```bash
+npm update @liveblocks/client@1.0 @liveblocks/node@1.0
+```
+
+The major release of 1.0 marks the maturity of Liveblocks. We added additional
+features to [Webhooks](/docs/guides/webhooks), enhancements to the `Room` class,
+and more. The primary change you should note is the transition to making the
+`userId` mandatory in the `authorize` option. This change is in line with our
+[new pricing model](https://liveblocks.io/pricing).
+
+Let’s take a look at the changes below!
+
+## Changes in @liveblocks/client [#client]
+
+### Adds throttle option to createClient [#throttle]
+
+You can now specify the `throttle` option in
```

Comment:
If I'm not mistaken, it was already possible to specify the throttle value in the createClient option. The difference is that it's now possible to go as low as 16ms.

Reply:
True!

Reply:
@ofoucherot is right. The change here is that you can go as low at 16ms, meaning you can theoretically get 60 FPS updates. Great for live cursors!

@CTNicholas had some excellent tweets about this.

Can we please change the heading and description to reflect that?

Reply:
Here's the [tweet](https://twitter.com/ctnicholasdev/status/1622656493769359360), previously you could go as low as 80

Reply:
Thanks @CTNicholas I added the tweet to the guide,  as well as link to example.